src/ghdcbot/engine/notifications.py

```
@@ -365,6 +591,134 @@ def _sanitize_discord_pr_title(title: str) -> str:
     return text
 
 
+DEFAULT_CODERABBIT_BOT_LOGINS: tuple[str, ...] = ("coderabbitai", "coderabbitai[bot]")
+
+
+def _format_actor_mention(
+    storage: Storage | None,
+    github_user: str,
+    coderabbit_bot_logins: Iterable[str] | None = None,
```

Ruff (0.16.2)

[error] Undefined name `Iterable`

(F821)